app/build.gradle

```diff
@@ -44,6 +44,12 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    lintOptions {
+        disable "UnsafeExperimentalUsageError",
+                "UnsafeExperimentalUsageWarning",
+                "LogNotTimber"
```

Comment:
Nice change; we don't need to worry about the timber warnings that are useless to us.